src/main/java/net/sf/jabref/exporter/BibDatabaseWriter.java

```diff
@@ -208,162 +197,60 @@ public SaveSession savePartOfDatabase(BibDatabaseContext bibDatabaseContext, Lis
                         entryType -> typesToWrite.put(entryType.getName(), entryType));
```

Comment:
I think this is double code: First you check ! present and then you check the same with ifPresent..